mote/fedmsg_consumer.py

```diff
@@ -13,8 +14,21 @@
 # with this program; if not, write to the Free Software Foundation,
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #
+import time
+
+import fedmsg
 
 import soke
 
-# Currently a placeholder
-pass
+target = '.meetbot.meeting.complete'
+
+def main():
+    for _, _, topic, msg in fedmsg.tail_messages():
+        if not topic.endswith(target):
+            continue
+
+        print("A meeting just ended!  Let's sleep for 2s to dodge a race.")
```

Comment:
Oh, I thought an `import from __future__` was needed to obtain `print()`, although I could be wrong.